src/content/eth/index.md

```diff
@@ -16,7 +16,9 @@ lang: en
 
 Ether, or ETH, is Ethereum's native currency. It is "digital money" that can be sent over the internet instantly and cheaply, and also be used in many [Ethereum-based applications](/en/dapps/).
 
-The easiest way to get ETH is to [buy some](https://docs.ethhub.io/using-ethereum/how-to-buy-ether/). There are many cryptocurrency exchanges that will allow you to buy ETH, but the one you should use will depend on where you live and how you want to pay.
+The easiest way to get ETH is to buy some from an exchange or [wallet](/wallets). Availability differs from country to country so check where you can buy ETH based on where you live first.
```

Comment:
Here I'd probably recommend a text link to the /get-eth/ page, since we mention "The easiest way to get ETH".

Perhaps:
```
The easiest way to get ETH is to [buy ETH from an exchange or wallet](/get-eth/)
```